src/os/bluestore/BlueStore.cc

```diff
@@ -13786,7 +13786,7 @@ void BlueStore::_wctx_finish(
     auto& r = lo.r;
     txc->statfs_delta.stored() -= lo.e.length;
     if (!r.empty()) {
-      dout(20) << __func__ << "  blob release " << r << dendl;
+      dout(20) << __func__ << "  blob1 " << *b << " release " << r << dendl;
```

Comment:
@ifed01 i understand that you want to print out `*b`, but why s/blob/blob1/ ?

Reply:
@tchaikov - just a typo. Fixed!